### PR TITLE
Control-Systems

### PR DESCRIPTION
**Pull request by Udbhav17**
Added a .md file named Control-Systems inside a folder of the same name. All explanations for the topic along with resources are contained in the same file.

### CODE REVIEW

**Review by tharun634 — CHANGES_REQUESTED** [review]
Add an entry for `Control Systems` to the root `README.md`

File: Control-Systems/Control-Systems.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Control Systems
```
Comment (tharun634) [inline Control-Systems/Control-Systems.md]: Rename the folder to `Control_Systems`.
Rename the file to `README.md`

File: Control-Systems/Control-Systems.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+A transfer function represents the relationship between the output signal of a control system and the input signal, for all possible input values.
+In a Laplace Transform, if the input is represented by R(s) and the output is represented by C(s), then the transfer function will be: 
+![image](https://user-images.githubusercontent.com/72137415/148949693-fe4092f8-df35-4be8-a756-7f940c17aa7b.png)
+
+### Representation of Control Systems
+Control systems are represented in two main ways, which are as follows:
+   * Block Diagrams : Here, a control system is represented by blocks to depict it pictorially. Block diagrams contain blocks (which represent the transfer function and have one input and one output), take-off point (representing branching of signals) and summation points (which depict an algebraic summation of two or more input signals to one output signal).
+   ![image](https://user-images.githubusercontent.com/72137415/148892599-cfa0f187-9c33-4378-ac39-333fe9a0d379.png) \
+   More on block diagrams can be read here:
+   https://www.javatpoint.com/control-system-block-diagram \
```
Comment (tharun634) [inline Control-Systems/Control-Systems.md]: Convert all the links inside the content (excluding the Resources section) to hypertext. Like [this](https://www.javatpoint.com/control-system-block-diagram)

File: Control-Systems/Control-Systems.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+### Transfer Function
+A transfer function represents the relationship between the output signal of a control system and the input signal, for all possible input values.
+In a Laplace Transform, if the input is represented by R(s) and the output is represented by C(s), then the transfer function will be: 
+![image](https://user-images.githubusercontent.com/72137415/148949693-fe4092f8-df35-4be8-a756-7f940c17aa7b.png)
+
+### Representation of Control Systems
+Control systems are represented in two main ways, which are as follows:
+   * Block Diagrams : Here, a control system is represented by blocks to depict it pictorially. Block diagrams contain blocks (which represent the transfer function and have one input and one output), take-off point (representing branching of signals) and summation points (which depict an algebraic summation of two or more input signals to one output signal).
+   ![image](https://user-images.githubusercontent.com/72137415/148892599-cfa0f187-9c33-4378-ac39-333fe9a0d379.png) \
```
Comment (tharun634) [inline Control-Systems/Control-Systems.md]: Leave a gap between the image and text, the formatting at this place looks odd.
image

**Review by tharun634 — CHANGES_REQUESTED** [review]

**Comment by Udbhav17** [comment]
All changes made as asked

**Review by tharun634 — APPROVED** [review]
LGTM 👍

**Pull request merged**